x-pack/plugins/apm/public/components/routing/apm_route_config.tsx

```diff
@@ -59,6 +74,42 @@ const apmRoutes = {
       </Breadcrumb>
     ),
     children: {
+      // this route fails on navigation unless it's defined before home
+      '/service-groups': {
+        element: (
+          <Breadcrumb
+            title={ServiceGroupsBreadcrumnbLabel}
+            href={'/service-groups'}
+          >
+            <ApmMainTemplate
+              pageTitle={ServiceGroupsTitle}
+              environmentFilter={false}
+              showServiceGroupSaveButton
+            >
+              <ServiceGroupsRedirect>
+                <ServiceGroupsList />
+              </ServiceGroupsRedirect>
+            </ApmMainTemplate>
+          </Breadcrumb>
+        ),
+        params: t.partial({
```

Comment:
I would think you'd want to use t.type here, not t.partial, as some of the query params are required

Reply:
Thanks i missed this.